colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -257,8 +263,14 @@ class GeminiPlugin(DPPluginBase):
 
     def __init__(
         self,
+        chunk_config_dict: Optional[dict] = None,
         chunk_init_device: Optional[torch.device] = None,
-        placement_policy: str = "cpu",
+        placement_policy: str = "static",
+        shard_param_frac: float = 1.0,    # only for static placement
+        offload_optim_frac: float = 0.0,    # only for static placement
+        offload_param_frac: float = 0.0,    # only for static placement
+        warmup_non_model_data_ratio: float = 0.8,    # only for auto placement
+        steady_cuda_cap_ratio: float = 0.9,    # only for auto placement
```

Comment:
There are comments to state which placement this arg is for. You should include the comments to the docstring as well.